packages/dara-core/js/types/utils.tsx

```diff
@@ -18,8 +18,8 @@ import {
 export function isVariable<T>(variable: AnyVariable<T> | T): variable is AnyVariable<T> {
     return (
         variable &&
-        Object.keys(variable).includes('uid') &&
-        Object.keys(variable).includes('__typename') &&
+        variable.hasOwnProperty('uid') &&
+        variable.hasOwnProperty('__typename') &&
```

Comment:
For extra speed you can probably also add a check of the type, i.e.

`variable && typeof variable == 'object' && variable.hasOwnProperty...`

Reply:
Good point, updated